sdk/ai/ai-projects/review/ai-projects.api.md

```diff
@@ -38,10 +42,12 @@ export interface AgentOutput {
     model: string;
     name: string | null;
     object: "assistant";
-    response_format?: AgentsApiResponseFormatOptionOutput | null;
+    // Warning: (ae-forgotten-export) The symbol "AgentsApiResponseFormatOptionOutput_2" needs to be exported by the entry point index.d.ts
+    response_format?: AgentsApiResponseFormatOptionOutput_2 | null;
```

Comment:
Are we planning to update all of these to camelCase as well? `responseFormat` etc? 
```suggestion
    responseFormat?: AgentsApiResponseFormatOptionOutput_2 | null;
```

Reply:
Yes, as there are other references, it will be shows as _2, Final PR will not have it.